official/resnet/keras/keras_imagenet_main.py

```diff
@@ -146,7 +147,8 @@ def run(flags_obj):
       num_epochs=flags_obj.train_epochs,
       parse_record_fn=parse_record_keras,
       datasets_num_private_threads=flags_obj.datasets_num_private_threads,
-      dtype=dtype)
+      dtype=dtype,
+      drop_remainder=flags_obj.enable_xla)
```

Comment:
Please define a `drop_remainder` boolean (instead of directly using `flags_obj.enable_xla`); and for that variable add a comment explaining why drop_remainder and XLA are related.

Reply:
Done.